images/linux/scripts/installers/azpowershell.sh

```diff
@@ -7,6 +7,26 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
+# Install Azure CLI (instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+if isUbuntu20 ; then
+    sudo pwsh -Command 'Save-Module -Name Az -LiteralPath /usr/share/az_3.8.0 -RequiredVersion 3.8.0 -Force'
+
+    # Run tests to determine that the software installed as expected
+    echo "Testing to make sure that script performed as expected, and basic scenarios work"
+    if ! pwsh -Command '$actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_3.8.0:" + $env:PSModulePath;
+        if (!(get-module -listavailable -name Az.accounts)) {
+            Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
+        }
+        $env:PSModulePath = $actualPSModulePath'; then
+        exit 1
+    fi
+
+    # Document what was added to the image
+    echo "Lastly, documenting what we added to the metadata file"
+    DocumentInstalledItem "Az Module (3.8.0)"
+    exit 0

```

Comment:
Am I right that you want to keep Azure 3.8.0 only on Ubuntu 20.04? I worry that some AzDO tasks won't work properly after that.

Reply:
Yeah, only the latest. Is there any way to check?

Reply:
Not sure. Tests look green.
@alepauly , is there anyone who can help us to clarify it?

Reply:
I wouldn't worry too much, lets go for it and see what workflows out there say. This image version starts at preview only.

Reply:
Let's settle on the last version then